app/controllers/book_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BookController < ApplicationController
+  include Wor::Paginate
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def index
+    render_paginated Book.all, limit: 2
```

Comment:
I do not understand why you choose a limit of 2?

Reply:
I was just testing how paginate worked and forgot to take it out.